src/mongo/connectToMongoClient.ts

```diff
@@ -15,6 +15,11 @@ export async function connectToMongoClient(connectionString: string, appName: st
         useNewUrlParser: true
     };
 
+    if (!!connectionString.match(/^mongodb:\/\/(localhost|127\.0\.0\.1)/)) {
```

Comment:
nit: I prefer `.test` over `.match` and I almost always include the `i` flag to be safe
```suggestion
    if (/^mongodb:\/\/(localhost|127\.0\.0\.1)/i.test(connectionString)) {
```